app/matplotlib.ipynb

```diff
@@ -194,21 +265,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "labels = [\"Product A\", \"Product B\", \"Product C\", \"Product D\"]\n",
     "market_share = [30, 20, 15, 35]\n",
-    "\n",
-    "# write your code here"
+    "fig, ax = plt.subplots()\n",
+    "ax.pie(market_share, labels=labels, autopct='%1.1f%%', startangle=90)\n",
```

Comment:
The pie plot is missing a title. Consider adding a title to the pie plot to fulfill the task requirements.